code/game/atoms_movable.dm

```diff
@@ -109,6 +202,8 @@
 			O.Check()
 	if (orbiting)
 		orbiting.Check()
+	if(isturf(loc) && forced_gravity_by_turf && (loc != forced_gravity_by_turf))
+		forced_gravity_by_turf.reset_forced_gravity_atom(src)

```

Comment:
it's an error by itself if this happens, as i'm pretty sure I made the change happen in a turf's entered/exited.